packages/sanity/src/core/form/studio/FormBuilder.tsx

```diff
@@ -159,8 +161,8 @@ export function FormBuilder(props: FormBuilderProps) {
     [Input],
   )
   const renderField = useCallback(
-    (fieldProps: Omit<FieldProps, 'renderDefault'>) => <Field {...fieldProps} />,
-    [Field],
+    (fieldProps: Omit<FieldProps, 'renderDefault'>) => <Field {...fieldProps} version={version} />,
```

Comment:
wdyt if instead of adding it to the field we add it to the form provider and we make it accessible for all the components inside the form

Reply:
I think that could be a good idea. Definitely not committed to the current approach... this is my first pass at tracing the component tree and figuring out what to do 🙂. I'll take a look now that this is working. Thank you for the idea!

Reply:
I've made this change. Thank you 🙌.